src/components/visualizations/chart/chartOptionsBuilder.ts

```diff
@@ -278,47 +260,6 @@ export function findParentMeasureIndex(afm: AFM.IAfm, measureItemIndex: number):
     return -1;
 }
 
-export interface IPointData {
-    y: number;
-    x?: number;
-    z?: number;
-    value?: number;
-    format: string;
-    marker: {
-        enabled: boolean;
-    };
-    name?: string;
-    color?: string;
-    legendIndex?: number;
-    id?: string;
-    parent?: string;
-    drilldown?: boolean;
-    drillIntersection?: any;
-}
-
-export interface ICategoryParent {
-    name: string;
-}
-
-// since applying 'grouped-categories' plugin, 'category' type is replaced from string to object in highchart
-export interface ICategory {
-    name: string;
-    parent?: ICategoryParent;
-}
-
-export interface IPoint {
```

Comment:
are you sure this `IPoint` interface is not exported out of react-components ? that it could be already used by customer? please check also all modified interfaces in this PR that its not a part of public API :)

Reply:
Also be aware that you probably run into a lot of conflicts with merging this branch into 7.1.0  as you are renaming it here :)

Reply:
- About public interfaces, `iPoint` isn't listed here https://github.com/gooddata/gooddata-react-components/blob/master/src/index.ts#L77 and it is used in just 3 files. Then i think it safe to merge `IPoint` to `IPointData`.
- I have rebased with next-7.1.0 . There are some small conflicts and easy to solve. So it's ok.
-